react-app/src/components/GamePage.js

```diff
@@ -343,15 +344,24 @@ class Game extends Component {
 
   pointerDown(e) {
     if (e.target != null) {
-      this.selectedIndex = e.target.trashItemIndex;
-      this.selectedSprite = e.target;
-      this.dragStartScreenX = e.data.global.x;
-      this.dragStartScreenY = e.data.global.y;
+      let i = e.target.trashItemIndex;
 
-      this.dragStartObjectX = this.state.trashList[this.selectedIndex].x;
-      this.dragStartObjectY = this.state.trashList[this.selectedIndex].y;
+      let n = this.state.trashList.length;
```

Comment:
variable 'n' needs to be more descriptive.

Reply:
aye aye